pkg/lifecycle/terraform/when.go

```diff
@@ -0,0 +1,50 @@
+package terraform
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/replicatedhq/ship/pkg/api"
+	"github.com/replicatedhq/ship/pkg/state"
+	"github.com/replicatedhq/ship/pkg/templates"
+	"github.com/spf13/viper"
+)
+
+func (t *ForkTerraformer) evaluateWhen(when string, release api.Release) bool {
+	debug := level.Debug(log.With(t.Logger, "struct", "ForkTerraformer", "method", "execute"))
+
+	return evaluateWhen(when, release, debug, t.Viper, t.StateManager)
+}
+
+func (t *DaemonlessTerraformer) evaluateWhen(when string, release api.Release) bool {
+	debug := level.Debug(log.With(t.Logger, "struct", "DaemonlessTerraformer", "method", "execute"))
+
+	return evaluateWhen(when, release, debug, t.Viper, t.StateManager)
+}
+
+func evaluateWhen(when string, release api.Release, logger log.Logger, viper *viper.Viper, manager state.Manager) bool {
+	if manager == nil {
+		return true
+	}
+
+	builderBuilder := templates.BuilderBuilder{Logger: logger, Viper: viper, Manager: manager}
+
+	configState, err := manager.TryLoad()
+	if err != nil {
+		_ = logger.Log("terraform.when.loadState", err.Error())
```

Comment:
what's the reasoning behind assigning these to blank?

Reply:
Code style hints for "you're discarding an error!", which this code is doing

Explicit is better than implicit IMO